launcher/teeserver/tee_server_test.go

```diff
@@ -0,0 +1,64 @@
+package teeserver
+
+import (
+	"io"
+	"log"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"path"
+	"testing"
+
+	"github.com/google/go-tpm-tools/launcher/launcherfile"
+)
+
+func TestGetDefaultToken(t *testing.T) {
```

Comment:
The attestHandler should take a path to a default attestation token file. That way you don't have to create a file in a well-known location and risk race conditions (then you can remove the change in ci.yml). You can use https://pkg.go.dev/testing#T.TempDir

Reply:
good point, done